portal-frontend/src/app/features/applications/application-details/pofo-details/pofo-details.component.html

```diff
@@ -40,78 +40,86 @@
     ></app-no-data>
   </div>
 
-  <div class="grid-1"></div>
-  <div class="subheading2 grid-double">Fill to be Placed</div>
-
-  <div class="subheading2 grid-1">Volume</div>
-  <div class="grid-double">
-    {{ _applicationSubmission.soilToPlaceVolume }}
-    <span *ngIf="_applicationSubmission.soilToPlaceVolume">m<sup>3</sup></span>
-    <app-no-data [showRequired]="showErrors" *ngIf="!_applicationSubmission.soilToPlaceVolume"></app-no-data>
-  </div>
-
-  <div class="subheading2 grid-1">Area</div>
-  <div class="grid-double">
-    {{ _applicationSubmission.soilToPlaceArea }}
-    <span *ngIf="_applicationSubmission.soilToPlaceArea">m<sup>2</sup></span>
-    <app-no-data [showRequired]="showErrors" *ngIf="!_applicationSubmission.soilToPlaceArea"></app-no-data>
-  </div>
+  <div class="full-width soil-table">
+    <div></div>
```

Comment:
Same as [previous comment](https://github.com/bcgov/alcs/pull/1255/files#r1437797145)

Reply:
check previous reply, its same in all pages